src/main.ts

```diff
@@ -55,9 +55,13 @@ interface ReadwisePluginSettings {
   /** Should get any deleted books */
   refreshBooks: boolean,
 
-  /** Queue of books to refresh */
+  /** Queue of books to refresh.
+   * @deprecated LEGACY: only exists to preserve books that were deleted prior to v2.1.0 */
   booksToRefresh: Array<string>;
 
+  /** retryFailedBooks: queue of books to retry because of previous failure */
+  failedBooks: Array<string>;

```

Comment:
handles:
1. new deletion approach from #67
2. pre-existing bug wherein failed books would never resync if user did not have resync deleted books disabled